b0nbon/stack-overflow-lite
# Ft-162568293-user-able-view-posted-questions

*What does this PR do*  
Adds home page for the user

*Description of task to be completed*  
Create a user-friendly user interface with the following features:-

1. view posted questions

*How should this be manually tested*

1. Clone this repository
2. Cd into the Stack-OverFlow-Lite1 folder
3. Open index.html in your browser

*What are the relevant pivotal tracker stories*  
#162568293 A User should be able to see the landing page

*Screenshot*  
![home](https://user-images.githubusercontent.com/45699928/50039475-38122480-0044-11e9-916d-0e5b9e9d5450.JPG)
